bin/main.rb

```diff
@@ -1,109 +1,71 @@
 #!/usr/bin/env ruby
 # frozen_string_literal: true
 
-# This is the welcome page for the players
-puts ' Welcome to Tic Tac Toe'
-puts 'Please player one give us your name:'
-player1 = gets.chomp
-# The name of the first player is stored here
-puts 'Please player two give us your name:'
-player2 = gets.chomp
-puts "Please #{player1} choose between x or o"
-selection = gets.chomp.to_s
-# The name of the second player is stored here
-if selection == 'x'
-  puts "#{player1}, you are #{selection} for this game"
-  puts "#{player2}, you are o for this game"
-elsif selection == 'o'
-  puts "#{player1}, you are #{selection} for this game"
-  puts "#{player2}, you are x for this game"
-end
-# used selection to save whatever the value the first player chooses
-def board
-  puts
-  puts '1|2|3'
-  puts  '-----'
-  puts  '4|5|6'
-  puts  '-----'
-  puts  '7|8|9'
+# rubocop:disable Layout/LineLength, Lint/RedundantCopDisableDirective,Layout/LineLength
```

Comment:
Why are you disabling the line length cop? It is a good practice to keen your lines below 120 characters for readability.
```suggestion
```